### PR TITLE
feat(UI): add quotes when tests pass

### PR DESCRIPTION
**Pull request by JordanMooree**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #46178

@ahmadabdolsaheb This seems to function as expected but I have a few thoughts and questions:
- Would we want styling applied to the quote? It looks pretty basic if it's just a `p` tag.
- I used compliments instead of quotes from `motivation.json` since that was the way it was done in the legacy RWD. Would you rather see the quote? If the quote is preferred, should I add in the author as well? 
- Please provide feedback on any naming conventions or if there is another approach I should take on this

### CODE REVIEW

**Comment by JordanMooree** [comment]
It looks like the best way to address this is to put `<q>` tag around the quote so that [most browsers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q#browser_compatibility) will implement the text surrounded by quotation marks. More about [`<q>` tags here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q).

As for the success message, I have a few ideas:

- We can remove the text `Congratulations, your code passes.` and replace it with the quote 
- We can let go of the congratulations text altogether and only display a quote, a big check mark, and all other features that are soon to come maybe something like the legacy RWD:
image

Let me know what you think @ahmadabdolsaheb

**Comment by ahmadabdolsaheb** [comment]
Thanks for the suggestions: I am discussing the above scenarios on the related issue: https://github.com/freeCodeCamp/freeCodeCamp/issues/46178#issuecomment-1157678916.

Once we get some feedback, we can move forward with the implementation.

**Comment by ahmadabdolsaheb** [comment]
you could go with the following mock:
Screen Shot 2022-06-16 at 3 50 18 PM

The current congratulatory message could be set to sr-only.

**Comment by JordanMooree** [comment]
@ahmadabdolsaheb I've made the changes, any accessibility issues to address for these changes?

**Comment by ahmadabdolsaheb** [comment]
It looks good.  let's get this the progress bar in first so the completion state is more obvious. 
https://github.com/freeCodeCamp/freeCodeCamp/pull/46536/

**Comment by ahmadabdolsaheb** [comment]
@JordanMooree, the progress bar is open to contributors if you want to take it.

**Comment by JordanMooree** [comment]
@ahmadabdolsaheb sounds good, I'll take it 👍

**Comment by Sembauke** [comment]
Hey @JordanMooree,
Are you still working on this, otherwise we can open this for other contributors!

Have a nice day, and happy coding.

**Comment by JordanMooree** [comment]
Hey @Sembauke, Ahmad stated once the progress bar gets merged that this one should be good to go! 

I can get this pr rebased as soon as the progress bar gets merged

**Comment by raisedadead** [comment]
We are officially on Node.js 18 now. We have changed the merge requirements accordingly.

You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.

Thanks for your contribution.

**Comment by ahmadabdolsaheb** [comment]
We have a progress bar, so this should be ready to go.

File: client/src/templates/Challenges/classic/lower-jaw.tsx [hunk 1]
```diff
@@ -112,22 +127,22 @@ const LowerJawTips = ({
...
-        {congratulationText}
+        {text}
         {children}
       </p>
     </div>
   );
 };
 
+// const isBlockCompleted = 100;
+
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/lower-jaw.tsx:145]:
```suggestion
```

**Review by ahmaxed — APPROVED** [review]

**Review by raisedadead — CHANGES_REQUESTED** [review]
This being a new feature, we should add a test. In this case, a unit test would be enough. 


Can we go ahead and add a jest-based test, please?

**Comment by bbsmooth** [comment]
I think this is a cool concept and will make the steps much more interesting, but I am concerned about some of these quotes. I just got the following quote after passing a step:

`They're in another castle.`

Now maybe I'm showing my age here, but I have absolutely no idea how this quote relates to success. And looking through `motivation.json` I am finding many more quotes that I don't understand how they imply success.

Visually, I can see the checkmark icon next to the quote which is a tip-off that I passed the step. But if I am listening to the page then I will not know about the checkmark icon because we currently have it hidden from screen reader users. But now I am thinking that we should unhide it and give it an accessible name like "Step completed" so that screen reader users will get a clear message that they passed the step. Then, they would hear the following:

"Step completed. They're in another castle."

So if they are like me and have no idea what "They're in another castle." refers to, at least it will be clear up front that they successfully completed the step.

**Comment by raisedadead** [comment]
Oh wow! That is a good catch @bbsmooth, and yes, I agree that some of the quotes may not make sense to some of the users. But that should be for another issue/PR.

**Comment by ahmadabdolsaheb** [comment]
This is the latest state of the lower jaw quote, where it will live on a row of its own. closes https://github.com/freeCodeCamp/freeCodeCamp/issues/50634

Screenshot 2023-06-16 at 6 46 33 PM

**Comment by bbsmooth** [comment]
Functionally, LGTM.

I am noticing one small style discrepancy though:

style-issue

As you narrow the frame, the check mark icon gets smaller but the quote icon stays the same. I'm assuming the quote icon is the way we want it to work and the check mark icon needs to be fixed. Should we fix this in this PR?

**Comment by ahmadabdolsaheb** [comment]
I noticed that it is happening with the other two icons as well so it is independent of the quote icon.
Screenshot 2023-06-16 at 10 18 28 PM

We could address it in another issue/pr.

**Comment by JordanMooree** [comment]
Sorry for coming back a bit late to this - do we want to address this style issue in this PR?

**Comment by ahmadabdolsaheb** [comment]
it will be another pr since it was a pre existing issue. However, it would be great if you could add a cypress test to check look for the quote element when the code passes.

**Comment by JordanMooree** [comment]
Sounds good, I could add that.

**Comment by JordanMooree** [comment]
It's been a while since using GitHub haha (accidentally ran `git push -f`) here is the other PR #50798  I have opened @ahmadabdolsaheb

**Pull request closed without merging**